pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/util/ElasticSearchUtil.java

```diff
@@ -143,15 +147,17 @@ private static void initialize() {
         }
     }
 
-    public static synchronized void initClient() {
+    public static void initClient() {
         if (client == null) {
-            try {
-                client = getClientForElasticsearchCloud(DSMConfig.getSqlFromConfig(ApplicationConfigConstants.ES_URL),
-                        DSMConfig.getSqlFromConfig(ApplicationConfigConstants.ES_USERNAME),
-                        DSMConfig.getSqlFromConfig(ApplicationConfigConstants.ES_PASSWORD));
-            } catch (MalformedURLException e) {
-                throw new DsmInternalError("Error while initializing ES client", e);
-            }
+            initClient(DSMConfig.getSqlFromConfig(ApplicationConfigConstants.ES_URL),
+                    DSMConfig.getSqlFromConfig(ApplicationConfigConstants.ES_USERNAME),
+                    DSMConfig.getSqlFromConfig(ApplicationConfigConstants.ES_PASSWORD), getProxy());
+        }
+    }
+
+    public static synchronized void initClient(String url, String username, String password, String proxy) {
```

Comment:
Maybe some javadoc  and or a log entry that tells the caller about blank password implying local/test instance?